### ISSUE TITLE
[Usable] [Screen reader- Open Bot Dialog] Voiceover is announcing the name of check box and information multiple times.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1905]
**Actual Result:**
Voiceover is reading multiple times for checkboxes information present in the "Open bot dialog".
Voiceover announces “Open in Debug mode Open in Debug mode check box not checked” again and again.
1.Open in Debug mode 2. Azure for US Government

**Expected Result:**
Voiceover should announce the information only once i.e. the checkbox should be associated with the content and focus goes only once on the content.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot" button and activate it.
4. Navigate the dialog with voiceover.
5. Observe the issue.

**Note:**
1. Same issue observed for links present at "Create new bot configuration dialog".
2. Same issue observed with "Remove key value pair" dimmed button at "Add other service" dialog.
3. Same issue observed with "Browse" button present at "Settings".
4. Same issue observed for the check boxes present throughout the application such as "New bot configuration", "Add an endpoint for your Bot", "edit ngrok settings".

The same issue gets resolved for Win32, reference bug Id: [https://github.com/microsoft/BotFramework-Emulator/issues/1823](url)

**User Impact:**
Users who rely on voiceover will be affected to navigate the check boxes if the same info narrated multiple times.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Attachment:**
[Checkboxes are announced multiple times.zip](https://github.com/microsoft/BotFramework-Emulator/files/3689995/Checkboxes.are.announced.multiple.times.zip)

**Comment by tonyanziano** [comment]
I can't reproduce this behavior on the most recent Emulator build:

![checkboxes](https://user-images.githubusercontent.com/3452012/69365835-fae9e580-0c49-11ea-8a9e-38428e9426fa.gif)

Please reevaluate this scenario on the newest nightly release:

https://github.com/microsoft/botframework-emulator-nightlies/releases

**Comment by ShikhaMishra11** [comment]
@tonyanziano The issue still exist in latest build. You can repro this, in voice over mode (control+option+left/right key).

Please find the latest attachment. 
[Checkboxes are announced multiple times.zip](https://github.com/microsoft/BotFramework-Emulator/files/3882387/Checkboxes.are.announced.multiple.times.zip)

Test Environment:
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (version: 4.6.0-92058)

**Comment by ShikhaMishra11** [comment]
Verified and fixed .
